examples/storybook/src/stories/RatingMenu.stories.ts

```diff
@@ -0,0 +1,19 @@
+import { storiesOf } from '@storybook/angular';
+import { wrapWithHits } from '../wrap-with-hits';
+import meta from '../meta';
+
+storiesOf('RatingMenu', module)
+  .addDecorator(meta)
+  .add('default with panel header', () => ({
+    component: wrapWithHits({
+      template: `
+      <ais-panel header="Rating">
+        <ais-rating-menu
```

Comment:
We should use a different index to see an actual change inside the results.

Reply:
fixed here: 422062c